.pre-commit-config.yaml

```diff
@@ -1,5 +1,7 @@
+repos:
```

Comment:
I think this is supposed to come after the `fail_fast` line

Reply:
Yeah, that's correct.

Reply:
It's done